src/service/index.ts

```diff
@@ -56,6 +61,10 @@ export const discover = async () => {
         const ruleset = getRuleset(settings.ruleset);
         logger.info(`Using ruleset: ${ruleset.name}`);
 
+        if (settings.maxRequests !== undefined) {
+            logger.info(`Max requests: ${settings.maxRequests}`)
+        }
+
```

Comment:
This log statement should be moved at the beginning of the `processMovieResult` function, so it's also logged when running the smartRecommendation job.